thefuck/ui.py

```diff
@@ -20,6 +21,8 @@ def read_actions():
             yield const.ACTION_NEXT
         elif key in (const.KEY_CTRL_C, 'q'):
             yield const.ACTION_ABORT
+        elif key in ('\t'):
```

Comment:
This hasn't been added to the prompt, so the user doesn't know this is an option?